src/navigation/sectionalTabs/sectionalTabs.jsx

```diff
@@ -83,17 +87,22 @@ const defaultProps = {
     selectedTabKey: undefined,
     withContent: false,
 };
-
-const useStyles = (theme) => {
-    const borderColorContrastPrimary = get(theme, 'palette.border.contrastPrimary');
+const styles = (theme) => {
```

Comment:
NAB: Would love to keep the blank line in between 89 and 90 here if we can.

Reply:
Fixed.